js/data/bucket.js

```diff
@@ -176,9 +187,9 @@ Bucket.prototype.destroy = function(gl) {
     }
 };
 
-Bucket.prototype.trimBuffers = function() {
-    for (var bufferName in this.buffers) {
-        this.buffers[bufferName].trim();
+Bucket.prototype.trimArrays = function() {
```

Comment:
Nit: Can we call this `trimStructArrays` for consistency with `createStructArrays`?